applications/ShapeOptimizationApplication/python_scripts/custom_ios/wrl_io.py

```diff
@@ -40,6 +40,8 @@ def ReadModelPart(self, model_part):
 
         shapes = read_shapes(self.file_name)
 
+        node_usage_count = {}
```

Comment:
what about the following name: 
```suggestion
        node_multiplicity = {}
```

Requires also some adjustments further below.